roles/edpm_nodes_validation/tasks/main.yml

```diff
@@ -72,12 +72,13 @@
       ansible.builtin.command: hostname -f
       register: hostname
       changed_when: false
-
-    - name: Verify a FQDN is set
-      ansible.builtin.assert:
-        that: hostname.stdout.find(".") != -1
-        fail_msg: "{{ hostname.stdout }} does not contain . and does not appear to be an FQDN."
-        success_msg: "{{ hostname.stdout }} contains . and appears to be an FQDN."
+      retries: 5
+      delay: 60
+      until: hostname.stdout.find(".") != -1
+  rescue:
+    - name: Print the error
+      ansible.builtin.debug:
```

Comment:
It should be `.fail` I guess?

Now it will succeed and continue the normal execution... (Which I wonder what would lead to ~ why we need that verification of FQDN?)